source/Platforms.hx

```diff
@@ -0,0 +1,178 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.input.keyboard.FlxKey;
+import flixel.group.FlxGroup;
+
+/**
+ * ...
+ * @author Jess Geiger
+ */
+class Platforms extends FlxSprite 
+{
+    //set initial movement
+    private var UP:Bool = false;
+    private var DOWN:Bool = true;
+    private var LEFT:Bool = false;
+    private var RIGHT: Bool = true;
+    private var heavy:Float = 8000; // Declare higher gravity while on platform
+    private var blockSize:Int = 32; //Declare block size to be used in width and height
+    //Initialize min and max positioning
+    private var minX:Float;
+    private var maxX:Float;
+    private var minY:Float;
+    private var maxY:Float;
+    //Set starting position and width of platform
+    private var startX:Float;
+    private var startY:Float;
+    private var platformwidth:Int;
+    public var offsetX:Float = 0;
+    public var inContact:Bool = false;
+    private var platformVelocity:Int = 40;
+    public var touchingSprites:FlxTypedGroup<PlatformTracker> = new FlxTypedGroup<PlatformTracker>();
```

Comment:
You're right, that was a much simpler way to implement that. I thought since the sprites group added to PlayState.hx was a Typed Group that it would be easiest to keep the touchingSprites group as the same, but an array works much better.